README.md

```diff
@@ -107,12 +107,15 @@ The configuration can be added directly to the `config.xml` using the `<secretCo
 A secret file is made of JSON, and has the following data structure:
 
 ```json
-[
-  { "key": "foo", "value": "bar" }
-]
+{
+  "secret_key": "...",
+  "secrets": {
+    "my-password": "AES:..."
+  }
+}
 ```
 
-The secret defined in the above example can be used as: `{{SECRET:[Env1Secrets][foo]}}` and will have the value `bar` at run-time.
+The secret defined in the above example can be used as `{{SECRET:[Env1Secrets][my-password]}}` in environment variables or SCM materials.
```

Comment:
While, currently secret params are supported only in environment variables and SCM materials, we are working on adding support for more entities. Refer - https://github.com/gocd/gocd/pull/8503

Is it ok to change it to,
```
The secret defined in the above example can be used by defining a Secret Param with syntax: `{{SECRET:[Env1Secrets][my-password]}}` in entities which support secrets. For more information about supported entities refer the [Secrets Management](https://docs.gocd.org/current/configuration/secrets_management.html) documentation.```
```

Reply:
Updated, thanks!